src/main/java/frc/robot/commands/intake/SetIntakeSpeed.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.intake.IntakeSubsystem;
+
+/**
+ * This command sets the speed of the intake rollers.
+ */
+public class SetIntakeSpeed extends Command {
+
+  private IntakeSubsystem intakeSub;
+  private double speed;
+
+  /**
+   * Sets the speed of the intake rollers.
+   * 
+   * @param intakeSub The instance of the IntakeSubsystem class to be used.
+   * @param speed     Desired speed, from -1 to 1.
+   */
+  public SetIntakeSpeed(IntakeSubsystem intakeSub, double speed) {
+
+    this.intakeSub = intakeSub;
+    this.speed = speed;
+
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(intakeSub);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    intakeSub.setIntakeSpeed(speed);
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    intakeSub.setIntakeSpeed(0);
```

Comment:
It would be better to be very specific and have a method in the subsystem called `stopIntake()`. It would be very simple but would remove any possible confusion.